src/main/java/dev/jbang/cli/Info.java

```diff
@@ -253,9 +311,9 @@ public Integer doCall() throws IOException {
 				Object v = f.get(info);
 				if (v != null) {
 					if (v instanceof String || v instanceof Number) {
-						System.out.println(v);
+						out.println(v);
```

Comment:
I think that use the static import for System.out and then using the `out` here is little bit confusing that this out is that one from System package instead of a field.